src/sage/rings/polynomial/pbori/fglm.py

```diff
@@ -1,5 +1,5 @@
-from .PyPolyBoRi import Polynomial, BoolePolynomialVector
-from .pbori import FGLMStrategy, BooleSet
+from .pbori import BooleSet, FGLMStrategy
```

Comment:
We should avoid relative imports, no ?

Reply:
Yes, you are right. But I prefer to do this migration in a separate PR.